packages/NodeTypeResolver/TypeAnalyzer/MethodTypeAnalyzer.php

```diff
@@ -31,9 +30,6 @@ public function isCallTo(MethodCall $methodCall, string $expectedClass, string $
         return $this->isInstanceOf($methodCall->var, $expectedClass);
     }
 
-    /**
-     * @param non-empty-string $expectedName
-     */
```

Comment:
These should be kept

Reply:
Fixed https://github.com/rectorphp/rector-src/pull/2451/commits/02a4a551e7296d9f64b30c59576bb781d732d4a7